### PR TITLE
test helper functions

### PR DESCRIPTION
**Pull request by khalifan-kfan**
# Description

first tests for healper funtions

## Type of change

- [x] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)

## Trello Ticket ID
https://trello.com/c/GV45NGUr

## How Can This Been Tested?
pull the branch, run tests review the changed files

## Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] My changes generate no new warnings
- [ ] Any dependent changes have been merged and published in downstream modules

# Screenshots
not required

### CODE REVIEW

**Comment by khalifan-kfan** [comment]
still pushing these till 5%

**Review by rhodinemma — DISMISSED** [review]

File: src/helpers/validation.js [hunk 1]
```diff
@@ -89,9 +89,6 @@ export const validateProjectType = (projectType) => {
     return "Project Type should start with a letter";
   }
 
-  if (validateName(projectType) === "false_convention") {
```
Comment (neelxie) [inline src/helpers/validation.js:92]: @khalifan-kfan did you remove this function because we actually do use it.

File: package.json [hunk 2]
```diff
@@ -23,7 +23,7 @@
   "scripts": {
     "start": "react-scripts start",
     "build": "react-scripts build",
-    "test": "CI=true react-scripts test --coverage ",
+    "test": "react-scripts test --coverage -u --env=jsdom",
```
Comment (neelxie) [inline package.json:26]: @khalifan-kfan Did you forget to change this back, sir?

File: src/helpers/validation.js (hunk 1 above)
Comment (khalifan-kfan) [reply src/helpers/validation.js:92]: Alright sir

Comment (khalifan-kfan) [reply src/helpers/validation.js:92]: This condition is checked twice in the function @neelxie check line 84

**Review by neelxie — APPROVED** [review]
Good work on this.

**Pull request merged**